web/themes/new_weather_theme/templates/layout/page--point.html.twig

```diff
@@ -55,11 +55,26 @@
         </div>
         {% endif %}
 
-        <div class="wx-tab-container wx-focus-offset-205" id="current" role="tabpanel" aria-labelledby="current-conditions-tab-button" tabindex="0">
-          {{ drupal_block("weathergov_current_conditions") }}
-          {% include '@new_weather_theme/partials/radar.html.twig' with { 'point': weather.point  } %}
-
-          {% include '@new_weather_theme/partials/satellite.html.twig' %}
+        <div class="wx-tab-container wx-focus-offset-205" id="today" role="tabpanel" aria-labelledby="current-conditions-tab-button" tabindex="0">
+          <div class="grid-container">
```

Comment:
Let's add `padding-x-0 tablet:padding-x-2` here so that on mobile the cards take up the full width

Reply:
@eric-gade I think you may have missed this one